src/pudl/metadata/resources/ferc1.py

```diff
@@ -537,6 +537,40 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "retained_earnings_ferc1": {
+        "description": "Reatained Earnings - The structed part of schedule 118.",
+        "schema": {
+            "fields": [
+                "utility_id_ferc1",
+                "report_year",
+                "record_id",
+                "earnings_type",
+                "amount",
+                "starting_balance",
+                "ending_balance",
+                "balance",
+                "ferc_account",
+                "row_type_xbrl",
+            ],
+            "primary_key": [
+                "utility_id_ferc1",
+                "report_year",
+                "earnings_type",
+            ],
+        },
+        "sources": ["ferc1"],
+        "etl_group": "ferc1",
+        "field_namespace": "ferc1",
+    },
+    "retrained_earnings_appropriations_ferc1": {
```

Comment:
Also says retRained earnings here.

If this is "some of the structured part of schedule 118" does that mean that it's a subset of the table above which says it contains THE structured part of schedule 118?  I think probably not. So maybe both of these tables need descriptions that more specifically indicate **what** part of the retained earnings they are describing.

Reply:
the reason i pulled two of these tables through the extract step is because I wanted to look at the structured part of the table (which comes from just one xbrl table and is now associated with `retained_earnings_ferc1`) and one of the ballizion unstructured parts of the table. I chose the first one which are the appropriations of retained earnings.

This table is: one big table now called `retained_earnings_ferc1` which have little sub-sections inside of it with unstructured details pertaining to specific earnings types.

I think THE structured part of the table should be called retained earnings. I hope we can lump all of the unstructured parts together and call it `retained_earnings_details_ferc1`, but if we can't, the earnings types are pretty clear and easy to add as suffixes like I'm doing here with `retrained_earnings_appropriations_ferc1`

Reply:
I'm confused. Is there structured data in more than one of these tables? Or is there a single structured table and several unstructured tables?